### ISSUE TITLE
development issue and considering re-writing

### ISSUE THREAD

**Issue by adnanhf** [adnanhf/Recap-Processing-in-Excel#8]
Established core functions, any error on the program would be input's fault. It is time to start formatting cells. But I stumbled upon bordering the cell, cannot find a way to work around. I'm considering re-writing codes from the start.

**Comment by adnanhf** [comment]
wondering if column filtering would necessary or it's okay to just assigning values right away

```
#filtering data column
data = data.filter(items=filterlist)
del filterlist
```

**Comment by adnanhf** [comment]
I evaluated the execution time. Using column filtering would be "slower" than assigning values right away. I decided to delete the column filtering.

> execution time when using column filtering
![image](https://user-images.githubusercontent.com/16016956/145894159-0bc66756-2f3c-4522-9bfe-5d4b71d45ebc.png)

> execution time without column filtering
![image](https://user-images.githubusercontent.com/16016956/145894300-23e7b95b-f533-4737-a0cd-8bc813652a63.png)
